tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -1772,6 +1772,14 @@ def testSqrt(self):
 
 class RoundingTest(tf.test.TestCase):
 
+  def _compare_values(self, x, y=None, use_gpu=True):
```

Comment:
I think we want to test both the cpu implementation and the gpu implementation, no? Those are two different codes / devices, unless is controlled at the higher run levele with some env flag with two runs one for each configuration (CPU only, GPU only).